Source/Renderer/Context.js

```diff
@@ -263,6 +263,9 @@ define([
         ContextLimits._maximumViewportWidth = maximumViewportDimensions[0];
         ContextLimits._maximumViewportHeight = maximumViewportDimensions[1];
 
+        var highp = gl.getShaderPrecisionFormat(gl.FRAGMENT_SHADER, gl.HIGH_FLOAT);
```

Comment:
Take a closer look at this [code example](https://github.com/AnalyticalGraphicsInc/webglreport/blob/master/webglreport.js#L120).  We need to account for `HIGH_INT`, not just `HIGH_FLOAT`, even though it isn't a problem yet.  Let's add explicit getters for highp float and int, and then just check to see if either isn't support in `ShaderProgram`.